jcmagsay/jillianmagsaysay
# Code spliting components

### What's this PR do?
- Custom loader will enforce component based code splitting. 
- Creates component chunks
- Reduces payload by reducing dependencies for non-SPA apps
- Add PR template

### Where should the reviewer start?
`loaders/inject-global-scss.js`

### Does this add new dependencies? Is there an installation procedure? (bundle, bower, npm, etc)
Need to restart webpack `yarn start:dev` on local

### How should this be manually tested?
Review `<head>` to ensure that there are no repeated styles and all component styles are in stored in a `<style>` chunk

### Any background context you want to provide?
Will eventually add identifiers to style blocks to easily identify the chunks.

#### What gif best describes this PR or how it makes you feel?
![nurd](https://media.giphy.com/media/2wuWbwdFQlgkM/giphy.gif)
